.woodpecker/ansible.yaml

```
@@ -6,39 +6,36 @@ when:
 
 steps:
   - name: ansible (check)
-    image: woodpeckerci/plugin-ansible:pull_10
-    pull: true
+    image: woodpeckerci/plugin-ansible:0.2.1
     environment:
       ANSIBLE_ROLES_PATH: ./roles
     settings:
-      playbook: site.yaml
+      playbook: playbooks/run.yaml
       diff: true
       galaxy: requirements.yaml
       inventory: inventory
       syntax_check: false
-      limit: ci_server
       become: true
       check: true
       user: root
       galaxy_force: false
-      vault_password:
+      vault-password:
```

Was this change intended?

---

Was about to harmonize everything to `_`. Sanitation makes both work the same way.

---

In that case we should stick with `_` I guess?

```suggestion
      vault_password:
```